pages/now-showing.js

```diff
@@ -50,18 +50,22 @@ NowShowing.Chat = styled.div`
 `;
 
 NowShowing.getInitialProps = async () => {
-  if (process.env.NODE_ENV !== 'production') {
-    return { isLive: true };
-  }
+  let isStreamLive;
 
-  const baseUrl =
-    process.env.NODE_ENV === 'production'
-      ? 'https://clapcitycinema.herokuapp.com'
-      : 'http://localhost:3000';
-  const res = await fetch(`${baseUrl}/twitch?reqType=isLive`).then(async (response) => {
-    const reply = await response.json();
-    return reply.response;
+  console.log('reading: ', process.env.CONTENTFUL_ACCESS_TOKEN )
+
+  const client = require('contentful').createClient({
+    space: process.env.CONTENTFUL_SPACE_ID,
+    accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
+  });
+
+
+  await client.getEntries({
+    content_type: 'streamInfo',
+  }).then((res) => {
+    const { isLive } = [...res.items][0].fields;
+    isStreamLive = isLive;
   });
 
-  return { isLive: res };
+  return { isLive: isStreamLive };
```

Comment:
Wondering if we should just move our return line into the `.then` and the `.catch` block of the await above. In the `.then` case, we return the value we get from Contentful with ` return { isLive };` and in the `.catch` case we can return the backup values or something like that with a separate return call or error. That way we don't have to defined the broader scope `isStreamLive` variable and deal with having two variables that kinda mean the same thing (`isLive` and `isStreamLive`).